cmd/identities/get_test.go

```diff
@@ -63,10 +62,12 @@ func TestGetCmd(t *testing.T) {
 				return out
 			}
 			transform := func(token string) string {
-				if !encrypt {
-					return token
+				if encrypt {
```

Comment:
I assume this changed because we now, somewhere, properly check if the token can be decrypted and then this test failed?

Reply:
Exactly. I changed most tests across Kratos to actually use encryption, and this one was the only one to fail. Good sign :)